### PR TITLE
rubberband: Fix build for Linuxbrew

### PR DESCRIPTION
**Pull request by aswger**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
   I got error
```
==> Installing or updating 'bundler' gem
Error: Unable to resolve dependency: user requested 'did_you_mean (= 1.2.0)'
```
- [x] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.
   I got token error, don't know about the follow up to fix

-----

### CODE REVIEW

File: Formula/rubberband.rb [hunk 1]
```diff
@@ -29,6 +30,9 @@ def install
         "--disable-dependency-tracking",
         "--disable-silent-rules",
         "--prefix=#{prefix}"
+      system "make"
+      ENV['JAVA_HOME'] = Formula['openjdk'].opt_prefix
```
Comment (iMichka) [inline Formula/rubberband.rb]: Can you put this Linux-only code inside a “unless OS.mac? / end” block? This is useful for me when I merge changes from homebrew-core.

Comment (aswger) [reply Formula/rubberband.rb]: Hmm, it's already inside that block though. :cold_sweat:

Comment (iMichka) [reply Formula/rubberband.rb]: Right. I was reviewing this on my phone. Can you have a look at: https://github.com/Linuxbrew/homebrew-core/pull/11570

I would like to split the Linux/Mac blocks as in my PR, and remove the return statement. Can you adapt your code accordingly?

Comment (aswger) [reply Formula/rubberband.rb]:
```
      system "make"
      ENV['JAVA_HOME'] = Formula['openjdk'].opt_prefix
      system "make", "jni"
      system "make", "install"
```
`make` and `make jni` should be called before `make install`. Unless you do that you would get build error calling `make install`

**Comment by aswger** [comment]
@iMichka the pull request has been adapted from #11570

**Comment by iMichka** [comment]
Thanks @aswger ! This makes me gain precious time to work on other things. The bottle is not pushed.

I will try to build `ffmpeg` here: #11624
We can continue the discussion there.

**Pull request closed without merging**